libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -174,6 +174,22 @@ const AP_Param::GroupInfo AP_Proximity::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_FILT", 18, AP_Proximity, _filt_freq, 0.25f),
 
+    // @Param: _MIN
+    // @DisplayName: Proximity minimum range
+    // @Description: Minimum expected range for Proximity Sensor. Setting this to 0 will set value to manufacturer reported range.
```

Comment:
This comment doesn't *appear*to be true.  I can't see any assignment to `_min_m` corresponding to this comment...

Reply:
I might not be understanding PeterB's comment properly but I don't think we should ever assign _min_m to anything.  so we shouldn't be modifying parameters instead it should always be used as an input.  .. and I don't think this PR does change the parameter value ever.

@rishabsingh3003 can you confirm that this part of the description is correct?
`Setting this to 0 will set value to manufacturer reported range.`

Reply:
I think this line means that it is true:
`if (!is_zero(frontend._max_m)) {`

Reply:
Well, no, that just applies the limits in the parameters.  It doesn't seem to have anything to do with the "manufacturer reported range".

Reply:
Ah ok, the "manufacturer reported range" part could be worded better.

For _MIN the description could be, "Minimum distance in centimeters that sensor can reliably read.  Values below this parameter value will be ignored"

For _MAX the description could be, ""Maximum distance in centimeters that sensor can reliably read.  Values above this parameter value will be ignored"

Reply:
> Ah ok, the "manufacturer reported range" part could be worded better.
> 
> For _MIN the description could be, "Minimum distance that sensor can reliably read. Values below this parameter value will be ignored"
> 
> For _MAX the description could be, ""Maximum distance that sensor can reliably read. Values above this parameter value will be ignored"

That documentation would appear to actually match the code, and given this is user-facing that's a good thing :-)  I *think* we still apply the driver-specified limits separately, and that *isn't* reflected in there, but meh.

Reply:
@peterbarker I might have it worded wrong as Randy suggests. All I meant was that if you set the parameter as "0", we'll use the readings coming in from the driver, without any filter. Each driver already has limits specified while it was being written, so that way we end up using the "manufacturer reported range". I'll be happy to raise another PR to make changes, please let me know if you want me to.